lib/run-tests/transpile.js

```diff
@@ -1,15 +1,30 @@
 "use strict";
 
-const babel = require("@babel/core");
+const { default: loadConfig } = require("@babel/core/lib/config");
+const { runSync } = require("@babel/core/lib/transformation");
```

Comment:
We shouldn't use internal API if possible. I think that if you use the public `loadOptions` and `transformSync` we will still get a good improvement because the resolved plugin/preset descriptors are cached.